src/asset-management/AaveManager.sol

```diff
@@ -178,7 +170,7 @@ contract AaveManager is IAssetManager, Owned(msg.sender), ReentrancyGuard {
             return 0;
         }
 
-        rTokenBalance = shares[aOwner][aToken].mulWad(_getExchangeRate(lAaveToken));
+        rTokenBalance = shares[aOwner][aToken] * lAaveToken.balanceOf(address(this)) / lTotalShares;
```

Comment:
Do we currently have the issue where pools will report a 1 wei loss if on adjustManagement/similar?

Reply:
for small amounts and if there are multiple `adjustManagement` + `getBalance` calls within one block, then yes. 

But as soon as some time pass and some interest accumulates there this "small" loss would not happen

Reply:
Okay, probably not much we can do unless we wanted to have some sort of deadband. Happy to leave as is